ports/glutin/window.rs

```diff
@@ -1190,47 +1190,48 @@ impl WindowMethods for Window {
     }
 
     /// Has no effect on Android.
-    fn set_cursor(&self, c: Cursor) {
+    fn set_cursor(&self, c: CursorKind) {
```

Comment:
We could rename the `c` arg to be more clear I think.

Reply:
Sure, that works.